src/main/java/uk/gov/hmcts/reform/managecase/service/NotifyService.java

```diff
@@ -0,0 +1,66 @@
+package uk.gov.hmcts.reform.managecase.service;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+import java.net.ConnectException;
+import java.util.List;
+import java.util.Map;
+import javax.validation.ValidationException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.retry.annotation.Backoff;
+import org.springframework.retry.annotation.Retryable;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.managecase.ApplicationParams;
+import uk.gov.service.notify.NotificationClient;
+import uk.gov.service.notify.NotificationClientException;
+import uk.gov.service.notify.SendEmailResponse;
+
+@Service
+public class NotifyService {
+
+    private final NotificationClient notificationClient;
+    private final ApplicationParams appParams;
+
+    @Autowired
+    public NotifyService(ApplicationParams appParams,
+                         NotificationClient notificationClient) {
+        this.notificationClient = notificationClient;
+        this.appParams = appParams;
+    }
+
+    @Retryable(value = {ConnectException.class}, backoff = @Backoff(delay = 1000, multiplier = 3))
+    public List<SendEmailResponse> senEmail(final List<String> caseIds,
+                                            final List<String> emailAddresses) throws NotificationClientException {
+        if (caseIds == null || caseIds.size() == 0) {
+            throw new ValidationException("At least one case id is required to send notification");
+        }
+
+        if (emailAddresses == null || emailAddresses.size() == 0) {
+            throw new ValidationException("At least one email address is required to send notification");
+        }
+
+        List<SendEmailResponse> emailNotificationResponses = Lists.newArrayList();
+        for (String caseId : caseIds) {
+            for (String emailAddress : emailAddresses) {
+                emailNotificationResponses.add(this.notificationClient.sendEmail(
+                    this.appParams.getEmailTemplateId(),
+                    emailAddress,
+                    personalisationParams(caseId),
+                    createReference(),
+                    this.appParams.getReplyToEmailId()
+                ));
+            }
+        }
+        return emailNotificationResponses;
+    }
+
+    private Map<String, ?> personalisationParams(final String caseId) {
```

Comment:
I think we are getting the name and surname of the solicitor from PRD. Why not contain name and surname (also maybe salutation/title) in personalization of email?

Reply:
for now there is no requirement to include any personalised information, it is still in discussions. If we get info on including name and surname etc from business then we will change this

Reply:
Ok. The change to get a list of notification request objects will make the design more extendible, if more personalised notifications are demanded later.